### PR TITLE
Api documentation

### PR DESCRIPTION
**Pull request by alexjota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +34,163 @@ manifoldjs <website-url> [-d <app-directory>] [-s <short-name>] [-p <platforms>]
...
+
+####creteFireFox(w3cManifestInfo, generatedAppDir)
+Returns a promise that generates the application for the FireFox platform.
+
+`w3cManifestInfo` is the manifest´s data in JSON format.
+
+`generatedAppDir` is the destination directory of the generated app.
+
+####createWindows10app(w3cManifestInfo, generatedAppDir)
+Returns a promise that generates Generates the application for the Windows 10 platform.
```
Comment (marcoscaceres) [inline README.md]: > that generates Generates

typo?

**Comment by kenchris** [comment]
creteFirefox ... should be createFirefox 
FireFox ...should be Firefox (but do you mean Firefox OS) ?

**Comment by marcoscaceres** [comment]
On Thursday, April 30, 2015, Kenneth Rohde Christiansen <
notifications@github.com> wrote:

> creteFirefox ... should be createFirefox
> FireFox ...should be Firefox (but do you mean Firefox OS) ?
>

It probably should be FxOS, specially if this is for "open web apps".
That's to not confuse with support for w3c manifest when that lands in
Desktop.

**Comment by marcoscaceres** [comment]
(by Desktop I mean Firefox Desktop)

**Pull request merged**